org-brain.el

```diff
@@ -3195,7 +3195,10 @@ LINK-TYPE will be \"brain\" by default."
               nil choice))))
     (let ((link (concat link-type ":"
                         (if (org-brain-filep choice) choice (nth 2 choice)))))
-      (push link org-link--insert-history)
+      (if (version-list-< (version-to-list (org-release)) '(9 3))
```

Comment:
I think you could do `(version< (org-release) "9.3")` instead of using `version-list-<`. A bit cleaner.

Reply:
The reason to use `version-list-<` is that it allows us to compare version as a list instead of a string, which decouples major version number from minor numbers compared to a string like `"9.3"`. But I guess it's not going to make any difference at this stage.